### DISCUSSION TITLE
mysqldump: Error 2020: Got packet bigger than 'max_allowed_packet' when dumping table xxx

### DISCUSSION THREAD

**Discussion by pczartoryski** [spatie/db-dumper#179] [Ideas]
As described in another Spatie library - https://github.com/spatie/laravel-backup/discussions/1600 - it would be nice to be able to pass max_allowed_packet parameter -> either to CLI command or in config/databases.php . Eventually it can be hard coded for max value to avoid backup fails.

**Comment by erikn69** [comment]
This is an agnostic db dumper, they can't add an especific mysql option, just use `addExtraOption`
